guides/bigquery-public-datasets/redirects.txt

```diff
@@ -0,0 +1,2 @@
+#https://docs.netlify.com/routing/redirects/
+/test   https://cube.dev/blog/google-bigquery-public-datasets
```

Comment:
@tenebricosa I guess `/test` should be updated to something else, as you mentioned in DMs, right?

Reply:
That's right! There will be all the old URLs that we have in the guide.